test/app-luatest/console_debugger_session_test.lua

```diff
@@ -1,7 +1,6 @@
 local t = require('luatest')
```

Comment:
commit "test: refactor dbgr test with it helper" breaks tests and next commit fixes it. please mention it in commit message

Reply:
Done:
>   This patch breaks the test without debugger readline support patch,
  that is next in the patchset.